### ISSUE TITLE
capa 7.0.1 for Windows appears to Windows Defender as a trojan.

### ISSUE THREAD

**Issue by RionEV** [mandiant/capa#2025]
### Description

Capa 7.0.1 for windows appears to be detected by Windows Defender as a trojan, of type Trojan:Win32/Vigorf.A or as Backdoor:Win32/Bladabindi!ml

### Steps to Reproduce

1. Download Capa 7.0.1 from the Releases page

**Expected behavior:**

I expect that the .zip archive is not immediately detected as a threat.

**Actual behavior:**

The .zip archive is immediately detected as a threat, without being run. Confirmed on two different systems.

### Versions

Capa 7.0.1

### Additional Information

I would appreciate confirmation as to whether this is a false positive, and how this might have happened.
The file was detected as a threat without being run or extracted.

**Comment by mr-tz** [comment]
I suspect the false positives stem from capa's packaging (via PyInstaller) and/or the embedded rules (containing unusual features such as strings related to the detections you've mentioned).

For what it's worth, the Linux (https://www.virustotal.com/gui/file/ddb02f6cc5001e65dd54670313c4fff8374c1c4fbc69aa270df312618c832c3a) and macOS (https://www.virustotal.com/gui/file/a9d68e26cf7c42dc9bacee34e7f8f0ca9d2b3a75a3edaf77a0e569dcdfd6a918) releases have no malicious detections.

**Comment by RionEV** [comment]
Your response encouraged me to review the Zenbox report again to make absolutely certain I didn't misread it; I was about to say "this doesn't seem to account for the URL", but on review I've realized that the Zenbox report only states that there are URLs stored in the application's data. 

The only thing that remains now is the suggestion that FTP credentials are at risk - that one, I don't understand. Would an antivirus engine claim to observe that behaviour if it is only matching PyInstaller's packaging and rules?

**Comment by mr-tz** [comment]
There are rules to detect the collection of FTP-related information. Looks like that's what is picked up here?

![2024-03-11_16-42-34_firefox](https://github.com/mandiant/capa/assets/17606537/5cca134b-baf6-4035-8d38-be3d7923e27c)

**Comment by RionEV** [comment]
RIght, I must have misunderstood; I thought "Dropped Info" was offloaded to a remote device, not installed locally. Thank you for clarifying the report, I greatly appreciate that. 

Not fully sure of this issue tracker's etiquette, so I think it best to allow you or another contributor to close the issue. Thank you again for your assistance.

**Comment by RionEV** [comment]
Actually, before the issue is closed, is there a false positive reporting process for Windows Defender and the other three engines that can be pursued to prevent this causing trouble for new users, or is this expected enough that it can be put perhaps into an FAQ for the program on this github project?

**Comment by sasobadovinac** [comment]
I reported this to Microsoft as a false positive on Feb 26 via https://www.microsoft.com/en-us/wdsi/filesubmission and the answer from their analysts was ...  

> Analyst comments:
> 
> We have determined that the files meet our criteria for malware. At this time the detection will remain in place.
> 
> More detailed information about the approach and criteria categories currently used by the Microsoft researchers are available here:
> https://docs.microsoft.com/windows/security/threat-protection/intelligence/criteria
> 
> Thank you for contacting Microsoft.

**Comment by mr-tz** [comment]
Thank you! I've also created an issue to add notes on this to our documentation for future reference.
